client/packages/invoices/src/Prescriptions/LineEditView/Footer.tsx

```diff
@@ -66,35 +37,17 @@ export const Footer: FC<FooterProps> = ({
               marginLeft="auto"
             >
               <DialogButton
-                variant="previous"
-                disabled={!hasPrevious}
+                variant="cancel"
+                // disabled={!hasPrevious}
                 onClick={() => {
-                  handleSave(() =>
-                    setNavigateTo(
-                      RouteBuilder.create(AppRoute.Dispensary)
-                        .addPart(AppRoute.Prescription)
-                        .addPart(String(invoiceNumber))
-                        .addPart(String(previous))
-                        .build()
-                    )
-                  );
+                  // TO-DO
```

Comment:
Maybe the same as close for now? Feels like this button should work?

Reply:
I guess it needs to bypass the ConfirmOnLeaving thing too? is it just a `resetDraft()` call?